src/App/components/ProfilePage/components/LastPhotos.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import {
+  LastPhotosComponent,
+  HeadingContainer,
+  LastPhotosHeading,
+  LastPhotosContainer,
+  LastPhoto,
+  LastPhotoImg
+} from "./LastPhotos.styles";
+import FirstImage from "../../../../assets/images/1.jpg";
+import SecondImage from "../../../../assets/images/2.jpg";
+import ThirdImage from "../../../../assets/images/3.jpg";
+import FourthImage from "../../../../assets/images/4.jpg";
+import FifthImage from "../../../../assets/images/5.jpg";
+import SixthImage from "../../../../assets/images/6.jpg";
+import SeventhImage from "../../../../assets/images/7.jpg";
+import EightImage from "../../../../assets/images/8.jpg";
+import NinthImage from "../../../../assets/images/9.jpg";
+
+const LastPhotos = () => {
+  return (
+    <LastPhotosComponent>
+      <HeadingContainer>
+        <LastPhotosHeading>Last Photos</LastPhotosHeading>
+      </HeadingContainer>
+      <LastPhotosContainer>
+        <LastPhoto>
+          <LastPhotoImg src={FirstImage} />
```

Comment:
U can make `LastPhoto` as a reusable component and have an object mapped with all the images in a constants folder. Loop through the object and map it using the reusable component.

Reply:
I know. The whole component is a placeholder, I will change that in the future when we have our backend data setup.

Reply:
Tbh all the stuff is placeholders for now. When I have the actual data I'll refactor.